src/App.js

```diff
@@ -1,21 +1,46 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+import { injectGlobal, ThemeProvider } from 'styled-components';
+import Home from './containers/Home';
+import Favorites from './containers/Favorites';
+import { HeaderNav, HeaderNavLink } from './components/Header';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
+injectGlobal`
+  body {
+    
+    margin: 0;
   }
-}
+  body, button{
+    font-family: Ubuntu;
+  }
+`;
+
+const theme = {
+  primary: {
+    background: '#000',
+    foreground: '#fff',
+  },
+  secondary: {
+    background: '#4965f3',
+    foreground: '#fff',
+  },
+};
+
+const App = () => (
+  <ThemeProvider theme={theme}>
+    <div className="App">
+      <HeaderNav>
+        <HeaderNavLink to="/">Trending gifs</HeaderNavLink>
+        <HeaderNavLink to="/favorites">Mis favoritos</HeaderNavLink>
+      </HeaderNav>
+
+      <Switch>
+        <Route exact path="/" component={Home} />
+        <Route exact path="/favorites" component={Favorites} />
+        <Redirect to="/" />
```

Comment:
It would be nice to send the object and let the user know what happened with those not found routes.